tests/test_filter.py

```diff
@@ -1,120 +1,137 @@
-import json
+import pytest
+from copy import deepcopy
 
-from echr.steps.filter import format_conclusion, find_base_articles, split_and_format_article, format_parties, \
-    merge_conclusion_elements, format_article, format_subarticle
+from echr.steps.filter import *
```

Comment:
The * import is not advised by Codacity and counts as code issue. Could you explicit the functions used in the tests please?

Also, we try to avoid merge commits. Could you rebase against develop without merge commit?